index.html

```diff
@@ -9222,6 +9222,240 @@ <h3 id="example-2">
             </dd>
           </dl>
         </section>
+        <section>
+          <h2>
+            Processing
+          </h2>
+          <p>
+            Dynamics compression can be implemented in a variety of ways. The
+            <a>DynamicsCompressorNode</a> implements a dynamics processor that
+            has the following characteristics:
+          </p>
+          <ul>
+            <li>Fixed look-ahead (this means that an
+            <a>DynamicsCompressorNode</a> adds a fixed latency to the signal
+            chain).
+            </li>
+            <li>Configurable attack speed, release speed, threshold, knee
+            hardness and ratio.
+            </li>
+            <li>Side-chaining is not supported
+            </li>
+            <li>The gain reduction is reported <em>via</em> the
+            <code>reduction</code> property on the
+            <a>DynamicsCompressorNode</a>
+            </li>
+            <li>The compression curve is has three parts:
+              <ul>
+                <li>The first part is the identify: \(f(x) = x\).
+                </li>
+                <li>The second part is the soft-knee portion, which MUST be a
+                monotonically increasing function.
+                </li>
+                <li>The third part is a linear function of equation \(f(x) =
+                \frac{1}{ratio} \cdot x \).
+                </li>
+              </ul>This curve MUST be continuously differentiable, and
```

Comment:
I think we need to write that it is allowed to have a function that is not continuously differentiable iff the knee is 0, which kind of makes sense, because the purpose of a hard knee is to have an hard edge.